tests/test_xsd.py

```diff
@@ -200,7 +240,7 @@ def test_typo_property_mk8s(testdir, properly_decorated_test_function):
 
     # Add another property element for the testcase.
     xml_doc.find('./testcase/properties/property').attrib['name'] = 'wrong_test_id'
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('mk8s')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd()))
```

Comment:
Since an external program is accessing this method it should be marked as public.